skbio/sequence/_sequence.py

```diff
@@ -1408,6 +1408,74 @@ def count(self, subsequence, start=None, end=None):
         return self._string.count(
             self._munge_to_bytestring(subsequence, "count"), start, end)
 
+    @experimental(as_of="0.4.2-dev")
+    def replace(self, where, character):
+        """Replace values in this sequence with a different character.
+
+        Parameters
+        ----------
+        where : 1D array_like (bool) or iterable (slices or ints) or str
+            Indicates positions in the sequence to replace with `character`.
+            Can be a boolean vector, an iterable of indices/slices, or a
+            string that is a key in `positional_metadata` pointing to a
+            boolean vector.
+        character: str
```

Comment:
`character : str or bytes`

Note the spaces around `:`, the docs don't render correctly otherwise.